src/components/Insights/tickets/ScalingIssueInsightTicket/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { SpanScalingBadlyInsight } from "../../types";
+import { InsightTicketProps } from "../types";
+import { InsightJiraTicket } from "../../InsightJiraTicket";
+import { intersperse } from "../../../../utils/intersperse";
+import { ReactElement, useContext } from "react";
+import { CodeLocations } from "../common/CodeLocations";
+import { CommitInfos } from "../common/CommitInfos";
+import { DigmaSignature } from "../common/DigmaSignature";
+import { useSpanDataSource } from "../common";
+import { getCriticalityLabel } from "../../../../utils/getCriticalityLabel";
+import { ConfigContext } from "../../../common/App/ConfigContext";
+import {
+  ScalingIssueAffectedEndpoints,
+  ScalingIssueDuration,
+  ScalingIssueRootCauses
+} from "../common/ScalingIssueCommon";
+
+export const ScalingIssueInsightTicket = (
+  props: InsightTicketProps<SpanScalingBadlyInsight>
+) => {
+  const config = useContext(ConfigContext);
+
+  const insight = props.data.insight;
+
+  const { commitInfos, isLoading, codeLocations } =
+    useSpanDataSource<SpanScalingBadlyInsight>(
+      props.data.insight.spanInfo,
+      props.data.insight
+    );
+
+  const renderDescription = () => {
```

Comment:
It looks like description could be extracted and resued for ScalingIssueInsightTicketRootCause, also 
the same for attachment and summary

Reply:
Did it
The code is much nicer now